src/pykeen/nn/representation.py

```diff
@@ -1224,7 +1235,7 @@ def __init__(
 
         # shape inference
         shape = ShapeError.verify(shape=combination.output_shape(input_shapes=[b.shape for b in base]), reference=shape)
-        super().__init__(max_id=max_id, shape=shape, unique=any(b.unique for b in base), **kwargs)
+        super().__init__(max_id=max_id, shape=shape, unique=all(b.unique for b in base), **kwargs)
```

Comment:
unit test for this?

Reply:
i mean it makes sense, i'm just wondering if this subtle bug caused some problems elsewhere

Reply:
`unique` makes a difference whenever there is stochasticity in the underlying representation, e.g., because of dropout.